User e2e test (#90)
GoogleCloudPlatform/cloud-run-anthos-reference-web-app: webui/cypress/README.md

## Prerequisite

### Setup firebase private key

> This is not needed if run the e2e test on Cloud Build,
> in which the Cloud Build service account already have permission configured.

Reviewer: ```suggestion
This is not needed if running the e2e test on Cloud Build,
when the Cloud Build service account already have permission configured.
```

> when the Cloud Build service account already have permission configured.

Reviewer 2: nit: should be "has permissions configured"
